src/AspNet.Security.OAuth.Trakt/TraktAuthenticationOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.Trakt.TraktAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Trakt
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="TraktAuthenticationHandler"/>.
+    /// </summary>
+    public class TraktAuthenticationOptions : OAuthOptions
+    {
+        public TraktAuthenticationOptions()
+        {
+            ClaimsIssuer = TraktAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(TraktAuthenticationDefaults.CallbackPath);
```

Comment:
This can be simplified to `CallbackPath = TraktAuthenticationDefaults.CallbackPath;` as there's an implicit conversion from `string` to `PathString`.